### PR TITLE
Fixed bug when static content deploys including static data from disa…

### PR DESCRIPTION
**Pull request by rganzhuyev**
…bled modules.

https://github.com/magento/magento2/issues/24666

### Description (*)
Fixed logic for css preprocessor to check if module is enabled before including its less / css resources to the generated static styles.

### Fixed Issues (if relevant)
1. https://github.com/magento/magento2/issues/24666

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=24772) <br/>All committers have signed the CLA.

File: lib/internal/Magento/Framework/Css/PreProcessor/Instruction/MagentoImport.php [hunk 1]
```diff
@@ -57,25 +58,33 @@ class MagentoImport implements PreProcessorInterface
...
+     * @param ModuleManagerInterface $moduleManager
      */
     public function __construct(
         DesignInterface $design,
         CollectorInterface $fileSource,
         ErrorHandlerInterface $errorHandler,
         \Magento\Framework\View\Asset\Repository $assetRepo,
-        \Magento\Framework\View\Design\Theme\ListInterface $themeList
+        \Magento\Framework\View\Design\Theme\ListInterface $themeList,
+        ModuleManagerInterface $moduleManager
```
Comment (sidolov) [inline lib/internal/Magento/Framework/Css/PreProcessor/Instruction/MagentoImport.php]: Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

**Review by sidolov — CHANGES_REQUESTED** [review]
Please, cover changes with automated tests

File: lib/internal/Magento/Framework/Css/PreProcessor/Instruction/MagentoImport.php (hunk 1 above)
Comment (rganzhuyev) [reply lib/internal/Magento/Framework/Css/PreProcessor/Instruction/MagentoImport.php]: done.

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi @rganzhuyev,
Could you review test failures and adjust code or tests itself (if they outdated)?

Also would be great to cover your changes with integration or unit test. Could you do that?

**Comment by rganzhuyev** [comment]
> Hi @rganzhuyev,
> Could you review test failures and adjust code or tests itself (if they outdated)?
> 
> Also would be great to cover your changes with integration or unit test. Could you do that?

Yeah, could review and adjust this weekend.

**Comment by VladimirZaets** [comment]
@rganzhuyev , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**